team_admin/kit_shipping.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Kit Shipping
+---
+
+# Kit Shipping
+
+This page provides details of how to ship a Student Robotics kit. Please only ship a Student Robotics kit if instructed to do so by the [Kit Logistics Coordinator](mailto:logistics@studentrobotics.org). If you have a kit that you need to ship to us please follow the instructions in the [Packaging](#packaging) section. Once you have your kit packaged, or are looking to get a kit collected from us to be delivered to you, please follow the instructions in the [Arranging a Courier](#arranging-a-courier) section.
+
+## Packaging
+When shipping a complete kit (rather than a single part) it should be shipped in the white 18l Really Useful Box it was provided in. Inside the box, all parts of the kit should either be placed into the jiffy bags that they were provided in or wrapped in bubble wrap. The batteries **must** be placed inside the charging bag provided with the kit. If you suspect that any batteries are damaged then do not ship them and contact <logistics@studentrobotics.org> immediately. Any remaining space in the box should be filled either with paper packaging (old Amazon parcels are a good source of this) or bubble wrap. Please do not use packing peanuts as these are messy and hard to dispose of.
+
+Once all of the parts of the kit are safely packed into the Really Useful Box please tape up the lid and handles as shown in the photo below (to be added).
+
+## Arranging a Courier
+Arranging for a courier to collect the kit/package and deliver it can be done quickly and easily via a range of websites. The sections below outline how this is done for a selection of couriers. For all of these couriers, they will come and collect the package from the collection address you specify.
+
+Since the kits contain lithium polymer batteries there are some restrictions around shipping. We have confirmed that the couriers listed below are happy with shipping packages containing the size and quantity of batteries provided in the kit. Detailed step-by-step instructions for other couriers will be added to this page in due course.
```

Comment:
This paragraph seems at odds with the information actually provided below -- it suggests that there are multiple couriers (which there aren't) and that step-by-step instructions are missing (yet they seem to be present).